lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -187,9 +188,9 @@ function dosomething_reportback_menu() {
       'file' => 'dosomething_reportback.admin.inc',
     );
   }
-  $items['rb/%reportback_file'] = array(
```

Comment:
I realized that the fid link to `rb/[fid]` in the Reportback File teaser is output here in Entity module's tpl: http://cgit.drupalcode.org/entity/tree/theme/entity.tpl.php?id=724e719#n36

Perhaps it makes to keep these path, and the default RB File path as `rb/[fid]`, but the page callback could load the relevant Reportback entity, displaying the Fid (the new way). Obv nothing to get in for this PR, but I think that could be a clean way to handle how the default URI for a Reportback File entity should behave. Any thoughts?